letsencrypt-auto-source/pieces/bootstrappers/deb_common.sh

```diff
@@ -100,6 +100,10 @@ BootstrapDebCommon() {
     # XXX add a case for ubuntu PPAs
   fi
 
+  if [ ! -z "$configure_cron" ]; then
+    cron="cron"
```

Comment:
Ah, good catch. I spent a lot of time researching how different cron packages got along on openSUSE, but didn't notice this case for debian. I'll check for `systemd-cron`!

Reply:
@joohoi do you know of a good way to check if `systemd-cron` or `systemd-cron-next` are installed? I looked into this a bit/tried to use those packages myself, and couldn't figure out a simple solution.

Reply:
The targeted distros (debian, rhel, suse) all use systemd.

Rather than trying to detect if `systemd-cron` or `systemd-cron-next` are installed, I think it would be better to detect if `systemd` is the init. If it is, then create a `certbot-renew.timer` and a `certbot-renew.unit` and don't do anything with cron. This will work on _any_ distro using systemd (archlinux, ubuntu, fedora, mint, etc, etc, etc), and then the cron editing can be used for legacy systems and systems that don't ship systemd (gentoo, the BSDs, etc).